### PR TITLE
DOCS: Specifying mouse position and delta behaviour

### PR DESCRIPTION
**Pull request by adrian-koretski-unity3d**
### Description

Added documentation in Mouse.cs to specify the independent behaviour between position and delta. This is to help clarify ISXB-819

### Changes made

Added a few comment lines in Mouse.cs 

### Testing

-

### Risk

Docs only change. No technical risk.

### Checklist

Before review:

- [ ] Changelog entry added.
    - Explains the change in `Changed`, `Fixed`, `Added` sections.
    - For API change contains an example snippet and/or migration example.
    - JIRA ticket linked, example ([case %<ID>%](https://issuetracker.unity3d.com/product/unity/issues/guid/<ID>)). If it is a private issue, just add the case ID without a link.
    - Jira port for the next release set as "Resolved".
- [ ] Tests added/changed, if applicable.
    - Functional tests `Area_CanDoX`, `Area_CanDoX_EvenIfYIsTheCase`, `Area_WhenIDoX_AndYHappens_ThisIsTheResult`.
    - Performance tests.
    - Integration tests.
- [ ] Docs for new/changed API's.
    - Xmldoc cross references are set correctly.
    - Added explanation how the API works.
    - Usage code examples added.
    - The manual is updated, if needed.

During merge:

- [ ] Commit message for squash-merge is prefixed with one of the list:
    - `NEW: ___`.
    - `FIX: ___`.
    - `DOCS: ___`.
    - `CHANGE: ___`.
    - `RELEASE: 1.1.0-preview.3`.

After merge:

- [ ] Create forward/backward port if needed. If you are blocked from creating a forward port now please add a task to ISX-1444.

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/Devices/Mouse.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public struct MouseState : IInputStateTypeInfo
         public static FourCC Format => new FourCC('M', 'O', 'U', 'S');
 
         /// <summary>
-        /// Screen-space position of the mouse in pixels.
+        /// Screen-space position of the mouse in pixels. Updated independently from delta using WM. At high framerates, certain frames might not receive any new position data.
```
Comment (ekcoh) [inline Packages/com.unity.inputsystem/InputSystem/Devices/Mouse.cs]: We need to remember this is a platform-agnostic API. I would suggest changing "Updated", which is given, to "May be updated independently from delta depending on the platform."

Then you may add to remarks: "On Windows, delta originates from RAWINPUT API." or leave that out.

Comment (ekcoh) [reply Packages/com.unity.inputsystem/InputSystem/Devices/Mouse.cs]: "If a delta is needed that correlates with position, compute it based on the last reported position".

File: Packages/com.unity.inputsystem/InputSystem/Devices/Mouse.cs [hunk 2]
```diff
@@ -32,7 +32,7 @@ public struct MouseState : IInputStateTypeInfo
         public Vector2 position;
 
         /// <summary>
-        /// Screen-space motion delta of the mouse in pixels.
+        /// Screen-space motion delta of the mouse in pixels. Updated independently from position using raw mouse input. At high framerates, certain frames might not receive any new delta data.
```
Comment (ekcoh) [inline Packages/com.unity.inputsystem/InputSystem/Devices/Mouse.cs]: Suggest changing this similarly.

**Review by ekcoh — APPROVED** [review]
LGTM

**Review by duckets — APPROVED** [review]
Approved with changes applied direct to branch.

**Pull request merged**